Sources/ApolloAPI/SchemaConfiguration.swift

```diff
@@ -59,7 +52,8 @@ extension SchemaConfiguration {
     guard let typename = object["__typename"] as? String else {
       return nil
     }
-    return objectType(forTypename: typename)
+    return objectType(forTypename: typename) ??
+    Object(typename: typename, implementedInterfaces: [])
```

Comment:
Is this to handle unknown types?

Reply:
Yes! We aren't handling them as far as we'd like to in the future (where we actually request the implemented interfaces), but for now this just gives parity to 0.x, because you should be able to store them in the normalized cache if you have enough information to normalize them.

This doesn't handle casting to other type scopes because we don't know about interfaces and unions.